src/main.js

```diff
@@ -27,8 +29,10 @@ import "./styles/styles.scss";
 
   const enableDimMode =
     config.get("enableDimMode") && config.get("enablePlugin");
-  toggleDimMode(enableDimMode);
 
+  toggleDimMode(enableDimMode);
+  toggleHiddenItems(config.get("showHiddenItems") && config.get("enablePlugin"));
+  toggleTokenConversion(config.get("convertTokenValues") && config.get("enablePlugin"));
```

Comment:
does it not work on refresh if you put this in the startMaejokTools function i mentioned?